install/kubernetes/helm/istio/charts/kiali/templates/clusterrolebinding.yaml

```diff
@@ -10,7 +10,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: kiali
+  name: kiali{{- if .Values.dashboard.readOnly }}-viewer{{- end }}
```

Comment:
I would rename this "viewOnlyMode" to be consistent with [the analogous Kiali Operator setting](https://github.com/kiali/kiali/blob/v0.18/operator/deploy/kiali/kiali_cr.yaml#L121-L125). Once we integrate the Kiali Operator with the helm charts, we'll want to maintain naming consistency here so it isn't confusing.